### PR TITLE
@mzikherman - Adds related artworks to artwork schema

### PR DESCRIPTION
**Pull request by broskoski**
This is a new (or maybe just reworked) endpoint that we are going to take a look at on the artwork page. Force PR forthcoming.

### CODE REVIEW

File: schema/artwork/index.js [hunk 1]
```diff
@@ -439,6 +439,16 @@ const ArtworkType = new GraphQLObjectType({
       manufacturer: markdown(),
       series: markdown(),
       meta: Meta,
+      related: {
+        type: new GraphQLList(Artwork.type),
+        args: {
```
Comment (mzikherman) [inline schema/artwork/index.js]: doesn't look like this allows for paging/size: https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/api/v1/related_endpoint.rb#L268-L277

and the index it reads: https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/models/search/cf_artwork_similarity_index.rb

**Comment by mzikherman** [comment]
Excited to see this on the front-end!

File: schema/artwork/index.js (hunk 1 above)
Comment (broskoski) [inline schema/artwork/index.js]: It seems to work though? https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/api/v1/related_endpoint.rb#L277

Comment (mzikherman) [inline schema/artwork/index.js]: oh oh oh you're right, my b

**Pull request merged**